algebra/src/curves/mnt4753/mod.rs

```diff
@@ -1,3 +1,11 @@
+//! [Coda's MNT4-753](https://codaprotocol.github.io/snark-challenge/MNT4753.html) curve of
+//! its MNT4/6 cycle for fully recursive SNARKs.
+//! Both base field as well as prime order subgroup are of size 753 bits.
+//!
+//! Supposed to satisfy a security level of 128 Bit, but a recent estimate in the manner of
```

Comment:
"a recent estimate", assumes the code is tagged with a date. I would replace with the date of the paper. Also, in general, we are calculating the security level of the various curves - maybe we should add in all of those comments also a date there, e.g. as of June 2020, it serves a security level of ... bits.

Reply:
The "recent estimate" refers to Aurore's note that she has sent us, but is not published anywhere. Maybe we should put a link to the file on ginger (but I am afraid, that this won't be a persistent one).